tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<ol><li>Same as above, write more descriptive prints e.g. <code>print("a + b = {}".format(a + b))</code>  </li><li>Mention that the brocasting follows the same numpy broadcasting rule as in <a href="https://numpy.org/doc/stable/user/basics.broadcasting.html" target="_blank">https://numpy.org/doc/stable/user/basics.broadcasting.html</a>.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>